api/src/main/kotlin/com/inner/circle/api/controller/ConfirmController.kt

```diff
@@ -0,0 +1,39 @@
+package com.inner.circle.api.controller
+
+import com.inner.circle.api.controller.dto.PaymentApproveDto
+import com.inner.circle.api.controller.dto.PaymentResponse
+import com.inner.circle.api.controller.request.PaymentApproveRequest
+import com.inner.circle.core.usecase.SavePaymentApproveUseCase
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/payment/v1")
```

Comment:
```suggestion
@PaymentV1Api
```
`RestController`, `RequestMapping` 둘 다 제거할 수 있을 것 같아요.